source/docs/zero-to-robot/step-2/index.rst

```diff
@@ -1,4 +1,15 @@
-Updating and Configuring Pneumatics Control Module and Power Distribution Panel
-===============================================================================
+Step 2: Installing Dependencies
```

Comment:
I don't like the term Dependencies here. Dependencies for what? How about just Installing Software?

Reply:
Sure